instructions/Ssh-keys.md

```diff
@@ -0,0 +1,24 @@
+## SSH
```

Comment:
Maybe we can call this file `openstack` and put all related info there for now

Reply:
is "openstack" folder a good way?

Reply:
@safronovD 
yes